### PR TITLE
Create db data init script

### PR DESCRIPTION
**Pull request by Merteg**
Delete alembic migrations and seedings. Create one script that create
tables and push start data.

### CODE REVIEW

**Comment by Merteg** [comment]
I am ready for your reviews.

File: server/db_data_init.py [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+from sqlalchemy import create_engine
+from sqlalchemy.sql.expression import insert
+
+from server.models import (Base, user, user_status, tag, token, event, event_history,
+                           event_status, event_tag, category, subscribe,
+                           feedback, gallery, role)
+
+
+engine = create_engine('postgresql://admin:1@localhost:5555/eventme',
+                        echo=True)
```
Comment (ykobuk) [inline server/db_data_init.py]: pep8: continuation line over-indented for visual indent.

File: server/db_data_init.py [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+        "is_deleted": False}
+    )
+subscribe_data = (
+    {"user_id": 4, "event_id": 1, "is_favorite": False},
+    {"user_id": 2, "event_id": 2, "is_favorite": True},
+    {"user_id": 5, "event_id": 2, "is_favorite": True}
+)
+
+conn = engine.connect()
+conn.execute(role.Role.__table__.insert(), role_data)
```
Comment (ykobuk) [inline server/db_data_init.py]: In my opinion, the loop 'for' will look better:
list_1 = [role.Role, tag.Tag, category.Category, user_status.UserStatus, event_status.EventStatus,
          user.User, event.Event, event_tag.EventTag, event_history.EventHistory, gallery.Gallery,
          feedback.Feedback, subscribe.Subscribe]

list_2 = [role_data, tag_data, category_data, user_status_data, event_status_data, user_data,
          event_data, event_tag_data, event_history_data, gallery_data, feedback_data, subscribe_data]

conn = engine.connect()
for i in range(0, len(list_1)):
    conn.execute(list_1[i].__table__.insert(), list_2[i])

File: server/db_data_init.py (hunk 1 above)
Comment (ykobuk) [reply server/db_data_init.py]: pep8: continuation line over-indented for visual indent.

**Review by ykobuk — CHANGES_REQUESTED** [review]

File: server/db_data_init.py [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+
+engine = create_engine('postgresql://admin:1@localhost:5555/eventme',
+                        echo=True)
+Base.metadata.create_all(bind=engine)
+
+role_data = (
+    {"role": "admin"},
+    {"role": "user"},
+    {"role": "moderator"}
+)
```
Comment (serge30) [inline server/db_data_init.py]: Please use list here. Common rule is next: if you have something like a struct/record/row with constant length but contains different types, use tuple. But for arrays of values of one type and variable size, please use lists.

File: server/db_data_init.py (hunk 2 above)
Comment (serge30) [reply server/db_data_init.py]: You can make it even better:
```
data = [
    (role.Role, role_data),
    (tag.Tag, tag_data),
    ...
]

for model_class, model_data in data:
    conn.execute(model_class.__table__.insert(), model_data)
```

File: server/db_data_init.py [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+                           event_status, event_tag, category, subscribe,
+                           feedback, gallery, role)
+
+
+engine = create_engine('postgresql://admin:1@localhost:5555/eventme',
+                        echo=True)
+Base.metadata.create_all(bind=engine)
+
+role_data = (
+    {"role": "admin"},
```
Comment (serge30) [inline server/db_data_init.py]: You may instantiate model directly here: `role.Role(role="admin")`. But it's up to you.

**Review by serge30 — CHANGES_REQUESTED** [review]
Also did you remove `alembic` dependency from `requirements.txt`?

File: server/db_data_init.py (hunk 2 above)
Comment (Merteg) [reply server/db_data_init.py]: I have one more idea. 
Can I save testing data into the class? Smth like `user.User.test_data = {...}`
Is it normal?

Comment (serge30) [reply server/db_data_init.py]: I'm not sure it's a good idea, i'd rather keep a testing data separated in this module.

**Comment by Merteg** [comment]
Add new changes, I decided to keep executing instead of using orm.

File: server/server/models/__init__.py [hunk 5]
```diff
@@ -55,6 +55,13 @@ def get_dbsession():
     return dbsession
 
 
+def init_tables(engine):
+    from server.models import (user, user_status, tag, token, event, event_history,
+                           event_status, event_tag, category, subscribe,
+                           feedback, gallery, role)
```
Comment (serge30) [inline server/server/models/__init__.py]: Please fix indentation. Lines should be aligned by fist symbol after `(`

**Review by serge30 — CHANGES_REQUESTED** [review]
One last minor comment

**Comment by Merteg** [comment]
I did this

**Review by serge30 — APPROVED** [review]

**Pull request merged**